design-proposals/persist-vmi-firmware-uuid.md

```diff
@@ -0,0 +1,108 @@
+# Overview
+This proposal introduces a mechanism to persist the firmware UUID of a Virtual Machine in KubeVirt.
+By storing the firmware UUID, we ensure that it remains consistent across VM restarts.
+which is crucial for applications and services that rely on the UUID for identification or licensing purposes.
```

Comment:
While this is correct, I don't think that's the most important part of this proposal, but rather that the firmware UUID to become, well, a "real" UUID (universally unique ID). Can you please rephrase?